src/main/frontend/components/page.cljs

```diff
@@ -1019,15 +1019,15 @@
          [:div.r.flex.items-center.justify-between
           [:div
            (ui/tippy
-            {:html  [:small (str (t :page/show-whiteboards) " ?")]
```

Comment:
The question mark symbol may vary depending on the language. I removed it, since it's not required in this case.